uefi/src/lib.rs

```diff
@@ -7,6 +7,26 @@
 //! important UEFI concepts. For more details of UEFI, see the latest [UEFI
 //! Specification][spec].
 //!
+//! # Minimal Example
+//!
+//! Minimal example for an UEFI application using functionality of the
+//! `uefi` crate:
+//!
+//! ```ignore
```

Comment:
With `ignore` there's a good chance the code will stop compiling over time (in fact currently it's missing a `use uefi::println`). It's tricky to get `no_std` code compiling in a doc test, so how about linking to a file that is already tested by CI? Example:

```rust
//! ```ignore
#![doc = include_str!("../../template/src/main.rs")]
//! ```
```

Reply:
Oh, great idea! Thanks